split.js

```diff
@@ -143,6 +156,7 @@ Split = function (ids, options) {
             pair.gutter = gutter;
         }
 
-        el.style.width = 'calc(' + options.widths[i] + '% - ' + options.gutterWidth / 2 + 'px)';
```

Comment:
Good question! When I wrote that code, I thought the answer was "yes". Now I think about it again, I am not sure.

There is certainly something strange about the dragging. My impression is that it has something to do with the snapping, but I don't understand the snapping code yet.